src/misc/manifest.webmanifest

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
maybe we should put this in `src/pwa` or something?

Reply:
I don't know what else would go in here though. I think it would only be this file?

For misc, I was thinking of other files like "robots.txt", "opensearch.xml", ... anything misc and top-level.

Reply:
I was curious what devsite does and I think they put it in a pwa directory but the only other thing in their is the icons.

I guess misc is fine ¯\_(ツ)_/¯